bandc_c+a.rb

```diff
@@ -34,33 +50,28 @@
   
   elsif game_type == "a"  
```

Comment:
So, classic and advanced games have almost same logic. Try to rewrite them in one method and give parameter with type of game. You may keep features of game types in hash, for example. Is there something like constants in ruby?